tests/test_transforms/test_timestamp/test_holiday_transform.py

```diff
@@ -84,33 +84,37 @@ def two_segments_simple_df_min(simple_constant_df_min: pd.DataFrame):
 
 
 def test_holiday_with_regressors(simple_ts_with_regressors: TSDataset):
-    simple_ts_with_regressors.fit_transform([HolidayTransform(out_column="holiday")])
-    len_holiday = len([cols for cols in simple_ts_with_regressors.columns if cols[1] == "holiday"])
-    assert len_holiday == len(np.unique(simple_ts_with_regressors.columns.get_level_values("segment")))
+    holiday = HolidayTransform(out_column="holiday")
+    new = holiday.fit_transform(simple_ts_with_regressors)
+    len_holiday = len([cols for cols in new.columns if cols[1] == "holiday"])
+    assert len_holiday == len(np.unique(new.columns.get_level_values("segment")))
 
 
 def test_interface_two_segments_daily(two_segments_simple_df_daily: pd.DataFrame):
     holidays_finder = HolidayTransform(out_column="regressor_holidays")
-    df = holidays_finder.fit_transform(two_segments_simple_df_daily)
+    ts = TSDataset(two_segments_simple_df_daily, freq="D")
```

Comment:
You should fix the fixture itself, now it should be `two_segments_simple_ts_daily `

Reply:
Fix all the fixtures used in these tests